ci/clone_request.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import nbformat
+
+
+def main(queries):
+    return requests.get(f"http://localhost:8888/api/bookstore/clone{queries}")
+
+
+if __name__ == "__main__":
+    res = main("?s3_bucket=nteract-notebooks&s3_key=published/whateverwewant.json")
+    print(res.json())
+    res = main("/?s3_bucket=nteract-notebooks&s3_key=published/whateverwewant.json")
+    print(res.json())
```

Comment:
I assume this only works with tokens off here?

Reply:
yes i mentioned this in the commit messages expanded version. I don't intend for this to be the longterm solution — it is just a way to rapidly prototype these things.